types/ej.reports.all/index.d.ts

```diff
@@ -0,0 +1,2164 @@
+// Type definitions for non-npm package ej.reports 4.1
+// Project: http://help.syncfusion.com/js/typescript
+// Definitions by: Syncfusion <https://github.com/syncfusion>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+/// <reference types="jquery" />
+
+/*!
+*  filename: ej.reports.all.d.ts
+*  version : 4.1.0.16
+*  Copyright Syncfusion Inc. 2001 - 2019. All rights reserved.
+*  Use of this code is subject to the terms of our license.
+*  A copy of the current license can be obtained at any time by e-mailing
+*  licensing@syncfusion.com. Any infringement will be prosecuted under
+*  applicable laws.
+*/
```

Comment:
@sandersn @RyanCavanaugh @DanielRosenwasser Do we allow the copyrights other than MIT for definitions?